### PR TITLE
Account management create account

### PR DESCRIPTION
**Pull request by shan-57blocks**

### CODE REVIEW

File: packages/huma-shared/src/services/IdentityServiceV2.ts [hunk 1]
```diff
@@ -248,6 +269,45 @@ const getHumaAccount = async (
...
+ * Update huma account.
+ *
+ * @param {string} networkType Network type.
+ * @param {boolean} isDev Is dev environment or not.
+ * @param {HumaAccount} humaAccount The Huma account.
+ * @returns {Promise<void>} Promise that returns void.
+ */
+const humaAccountUpdate = async (
+  networkType: NETWORK_TYPE,
+  humaAccount: Partial<HumaAccount>,
```
Comment (ljiatu) [inline packages/huma-shared/src/services/IdentityServiceV2.ts:303]: Should we use a more specific type here? `Partial<HumaAccount>` suggests that the ID could also be passed in as a request param, when in reality it cannot.

**Review by ljiatu — APPROVED** [review]
LGTM % comment

File: packages/huma-shared/src/utils/config.ts [hunk 2]
```diff
@@ -44,6 +44,14 @@ const getIdentityAPIUrl = (chainId: number, isDev = false) =>
     isDev,
   )}.identity-verification.huma.finance`
 
+const getIdentityAPIUrlV2 = (
+  networkType: 'testnet' | 'mainnet',
```
Comment (mliu) [inline packages/huma-shared/src/utils/config.ts:48]: use the NETWORK_TYPE enum you defined above?

**Review by mliu — APPROVED** [review]
Approving with comment, PTAL at Jiatu's comment as well

**Pull request merged**